src/frontends/onnx/frontend/src/op/multinomial.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/multinomial.hpp"
+
+#include "exceptions.hpp"
+#include "openvino/op/multinomial.hpp"
+#include "utils/common.hpp"
+
+using namespace ov::op;
+using ::ONNX_NAMESPACE::TensorProto_DataType;
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace op {
+namespace set_1 {
+
+ov::OutputVector multinomial(const ov::frontend::onnx::Node& node) {
+    const auto input = node.get_ov_inputs().at(0);
+
+    const auto sample_size = node.get_attribute_as_constant<int64_t>("sample_size", 1);
+
+    const auto dtype =
+        node.get_attribute_value<int64_t>("dtype",
+                                          static_cast<int64_t>(TensorProto_DataType::TensorProto_DataType_INT32));
+    const auto seed = node.get_attribute_value<float>("seed", 0.0f);
+    const auto target_type = common::get_ov_element_type(dtype);
+    const uint64_t global_seed = 0;
+    const auto seed_uint64 = static_cast<uint64_t>(seed * 1000);
+
+    auto multinomial_op =
+        std::make_shared<ov::op::v13::Multinomial>(input, sample_size, target_type, seed_uint64, global_seed);
```

Comment:
There is no such constructor for OV Multinomial, the "with_replacement" and "log_probs" flags need to be set true or false, (to be verified with onnx, but seems like both should be false) please refer to:
https://docs.openvino.ai/2024/api/c_cpp_api/classov_1_1op_1_1v13_1_1_multinomial.html

Let's try:
```suggestion
    auto multinomial_op =
        std::make_shared<ov::op::v13::Multinomial>(input, sample_size, target_type, false, false, seed_uint64, global_seed);
```
*Those "false" values can be assigned to some constexpr variables to make it more readable.

Reply:
Please verify correct values for those flags, that was an example, but "true", "true" can be valid as for TF:
https://github.com/openvinotoolkit/openvino/blob/b2507967bf659043a825458128e78ee4f1d12bbd/src/frontends/tensorflow_common/src/op/multinomial.cpp#L23